### PR TITLE
Fixing conflicts docs to reflect actual update/delete behavior

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

After viewing the code path, I have determined this is the current behaviour (line 213-221).

Discussion about whether this behaviour is good is out of scope for this change. This just seeks to match observed functionality.

For reference - JIRA ticket - https://enterprisedb.atlassian.net/browse/BDR-3876

### CODE REVIEW

**Review by drothery-edb — APPROVED** [review]

File: product_docs/docs/pgd/5/consistency/conflicts.mdx [hunk 1]
```diff
@@ -210,8 +210,15 @@ If the deleted row is still detectable (the deleted row wasn't removed by `VACUU
...
 
-Another type of conflicting `DELETE` and `UPDATE` is a `DELETE` that comes after the row was updated locally. In this situation, the outcome depends on the type of conflict detection used. When using the
-default, [origin conflict detection](#origin-conflict-detection), no conflict is detected, leading to the `DELETE` being applied and the row removed. If you enable [row version conflict detection](#row-version-conflict-detection), a `delete_recently_updated` conflict is generated. The default resolution for this conflict type is to apply the `DELETE` and remove the row. However, you can configure this or a conflict trigger can handled it.
+Another type of `UPDATE`/`DELETE` conflict is when a `DELETE` for a row comes after that row was updated locally. In this situation, the outcome depends on the type of conflict detection used and the replication identity in use. 
+
+When using the default, [origin conflict detection](#origin-conflict-detection) and only primary keys for replication, no conflict is detected, leading to the `DELETE` being applied and the row being removed.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/consistency/conflicts.mdx:215]:
```suggestion
When using [origin conflict detection](#origin-conflict-detection) (the default) and only primary keys for replication, no conflict is detected, leading to the `DELETE` being applied and the row being removed. 
```

File: product_docs/docs/pgd/5/consistency/conflicts.mdx [hunk 2]
```diff
@@ -210,8 +210,15 @@ If the deleted row is still detectable (the deleted row wasn't removed by `VACUU
...
-default, [origin conflict detection](#origin-conflict-detection), no conflict is detected, leading to the `DELETE` being applied and the row removed. If you enable [row version conflict detection](#row-version-conflict-detection), a `delete_recently_updated` conflict is generated. The default resolution for this conflict type is to apply the `DELETE` and remove the row. However, you can configure this or a conflict trigger can handled it.
+Another type of `UPDATE`/`DELETE` conflict is when a `DELETE` for a row comes after that row was updated locally. In this situation, the outcome depends on the type of conflict detection used and the replication identity in use. 
+
+When using [origin conflict detection](#origin-conflict-detection) (the default) and only primary keys for replication, no conflict is detected, leading to the `DELETE` being applied and the row being removed. 
+
+If `REPLICA IDENTITY FULL` is being used for replication, the conflict will be detected and a `delete_recently_updated` conflict is generated.
```
Comment (PJMODOS) [inline product_docs/docs/pgd/5/consistency/conflicts.mdx:217]: This is not accurate

Comment (PJMODOS) [reply product_docs/docs/pgd/5/consistency/conflicts.mdx:217]: The conflict is detected when REPLICA IDENTITY FULL is used and one of the following things are true:
- table uses row version conflict detection/resolution (this anyway requires REPLICA IDENTITY FULL)
- table uses per-column conflict detection/resolution aka CLCD (this anyway requires REPLICA IDENTITY FULL)
- table uses custom conflict resolution triggers

**Review by PJMODOS — CHANGES_REQUESTED** [review]
See comment inline

**Comment by djw-m** [comment]
I refer back to the original Jira ticket which reports that this isn’t the
case - if this is in fact a bug or user error, we would appreciate dev
triage before it gets to requests for documentation changes.

**Comment by josh-heyer** [comment]
Closing this unless / until someone wants to collaborate on it.

**Pull request closed without merging**